static/app/views/performance/onboarding.tsx

```diff
@@ -167,76 +173,81 @@ function Onboarding({organization, project}: Props) {
   }
 
   return (
-    <OnboardingPanel image={<PerfImage src={emptyStateImg} />}>
```

Comment:
Nothing here is changed except 176-178, the whole thing looks changed here just because of an extra tab 😢